tlatools/org.lamport.tlatools/src/tla2sany/drivers/SANY.java

```diff
@@ -437,30 +412,30 @@ private static void printUsage()
    * barring errors too severe, calls the Explorer tool with the resulting 
    * ExternalModuleTable.
    */
-  public static void SANYmain(String args[]) {
+  public static int SANYmain(String[] args) {
```

Comment:
this is an interface change. If needed I can create a new method to return int instead.